climakitae/util/utils.py

```diff
@@ -34,33 +34,6 @@ def downscaling_method_as_list(downscaling_method):
     return method_list
 
 
-def scenario_to_experiment_id(scenario, reverse=False):
```

Comment:
Isn't this function imported in the other files?

Reply:
For some reason we had two copies of it in this module! Not sure why. I just deleted the copy